packages/nimble-components/package.json

```diff
@@ -26,14 +26,14 @@
     "generate-scss:run": "node build/generate-scss/dist/index.js",
     "tdd": "npm run build-components && npm run test-chrome",
     "tdd:watch": "npm run build-components:watch & npm run test-chrome:watch",
-    "test-chrome:debugger": "karma start karma.conf.js --browsers=ChromeDebugging",
-    "test-chrome:verbose": "karma start karma.conf.verbose.js --browsers=ChromeHeadlessOpt --single-run",
-    "test-chrome:watch": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --watch-extensions js",
-    "test-chrome": "karma start karma.conf.js --browsers=ChromeHeadlessOpt --single-run",
-    "test-firefox": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run",
-    "test-webkit": "karma start karma.conf.js --browsers=WebkitHeadless --single-run",
-    "test-webkit:debugger": "karma start karma.conf.js --browsers=Webkit",
-    "test": "npm run test-chrome:verbose"
+    "test-chrome:debugger": "karma start karma.conf.js --browsers=ChromeDebugging --skip-tags failsChrome",
```

Comment:
ni / pref: I think the name Skip and capitalizing the tab names like `#SkipChrome` `#SkipFirefox` `#SkipWebkit`

Skip instead of Fails because there may be other reasons that the test is temporarily skipped, ie failure, intermittency, performance, etc.

Reply:
Changed to "Skip"